cmd/bootstrap/cmd/seal.go

```diff
@@ -43,11 +45,16 @@ func constructRootResultAndSeal(
 		RandomSource:       getRandomSource(flagBootstrapRandomSeed),
 	}
 
+	dkgPubKeys := make([]crypto.PublicKey, 0)
+	for _, participant := range dkgData.Participants {
+		dkgPubKeys = append(dkgPubKeys, participant.KeyShare)
+	}
+
```

Comment:
⚠️  This is iterating over a map (random order) to populate the list of public keys. The order of keys in `DKGParticipantKeys` is meaningful and must be preserved. 

```suggestion
	dkgPubKeys := make([]crypto.PublicKey, len(dkgData.Participants))
	for i, participant := range dkgData.Participants {
		dkgPubKeys[participant.Index] = participant.KeyShare
	}

```